### ISSUE TITLE
Missing add-mass coefficient for the monopile

### ISSUE THREAD

**Issue by kenloen** [IEAWindSystems/windIO#68]
The monopile is missing the added mass coefficient (Ca). It currently defines a `drag_coefficient`: https://github.com/IEAWindTask37/windIO/blob/main/windIO/turbine/IEAontology_schema.yaml#L1247C29-L1247C45
There should be an equivalent entry for the added mass coefficient. Maybe with a default of 1?!

**Comment by ptrbortolotti** [comment]
waiting for @dzalkind to comment

**Comment by luwang00** [comment]
I agree there should be an added mass coefficient. A default of 1 should be adequate for most cases.

**Comment by ptrbortolotti** [comment]
thank you @luwang00  and @kenloen , this commit bring in the distribution of Ca for the monopile d883d346c34566e38fc8165c2f708ea92dfe6274
